superset/utils.py

```diff
@@ -746,3 +746,9 @@ def get_filter_key(f):
                     form_data['filters'] += [filtr]
         # Remove extra filters from the form data since no longer needed
         del form_data['extra_filters']
+
+def unique_list(target_list):
```

Comment:
Don't think this helper is useful

Reply:
the easy way to do this is `list(set(list))` though you can usually just use a set from the get go and stick to it

Reply:
true didn't realise it was that simple 🥇